content/docs/contributing/getting_started.md

```diff
@@ -22,13 +22,17 @@ To contribute to the handbook you will need
   extended version with Sass/SCSS support)
 - Your favourite text editor
 
+Optionally:
+
+- [markdownlint](https://github.com/markdownlint/markdownlint) in
+  case you want to lint your branch before making a pull request.
```

Comment:
The colon here is inconsistent.

I'm not sure I like a list of one item.

Reply:
Thanks for the helpful comments. I've merged the two lists together.